package.json

```diff
@@ -63,7 +64,9 @@
   "devDependencies": {
     "babel-core": "6.26.0",
     "babel-loader": "7.1.2",
+    "babel-polyfill": "6.26.0",
```

Comment:
What are you polyfilling?

Reply:
In the commit I'm removing. Superceded by #1726